packages/next/client/index.tsx

```diff
@@ -674,6 +678,10 @@ async function doRender({
           el.removeAttribute('media')
           el.setAttribute('data-n-p', '')
         })
+
+      // Force browser to recompute layout, which prevents a flash of unstyled
+      // content:
+      getComputedStyle(document.body, 'height')
```

Comment:
@Timer This line causes crashes on our app:

> TypeError: Window.getComputedStyle: 'height' is not a valid pseudo-element

According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) only pseudo element selectors are allowed here. Do you remember what you tried to achieve here?